### ISSUE TITLE
[Bug][A11Y][Inputs] Add a hint to explain how to interact with input fields (text, password, pin-code)

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#495]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

iOS and Android

### App version

0.6.0

### Describe the issue

On Android : for all input components (text, password, pin code) add a hint to explain how to interact : "double tap to edit"

On iOS : do the same for password only, as it's already done for others

### Expected Behavior

.

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (607)
Version: 1.0.0
Build type: Alpha
Build details (GitHub): 548
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (607)
Version: 1.0.0
Build type: ALPHA
Build details (GitHub): 548
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
OK on 1.0.0 alpha (607)

**Comment by B3nz01d** [comment]
@nouha06 I cannot find build 607 on Android, where can this be tested?

**Comment by B3nz01d** [comment]
ok on iOS, but @nouha06 how can I test this as I do not see the build 607 on android

**Comment by nouha06** [comment]
@B3nz01d it’s available on [App Distribution](https://console.firebase.google.com/project/ouds-f91d8/appdistribution/app/android:com.orange.ouds.alpha.flutterapp/releases)

**Comment by B3nz01d** [comment]
@nouha06 you are missing the phone number input with this update, also the helper text is not read at all on this same component

- [ ] #571

**Comment by B3nz01d** [comment]
Finished during Sprint # 27
